client/mig/main.go

```diff
@@ -223,6 +230,20 @@ func main() {
 	for _, arg := range os.Args[1:] {
 		a.Name += arg + " "
 	}
+
+	if targetfound != "" && targetnotfound != "" {
+		panic("Both -target-foundanything and -target-foundnothing cannot be used simultaneously")
+	}
+	if targetfound != "" {
+		targetQuery := fmt.Sprintf(`id IN (select agentid from commands, json_array_elements(commands.results) as 
+			r where actionid=%s and r#>>'{foundanything}' = 'true')`, targetfound)
```

Comment:
we probably don't want a newline in here, as the string will be added to the json